src/coredump/coredump.c

```diff
@@ -63,6 +64,8 @@
 #define JOURNAL_SIZE_MAX ((size_t) (10LU*1024LU*1024LU))
 #endif
 
+
+
```

Comment:
spurious extra blank lines

Reply:
Addressed the comment. Please check once.